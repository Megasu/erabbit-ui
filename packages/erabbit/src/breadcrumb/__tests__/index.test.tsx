import { mount } from '@vue/test-utils'

import { Breadcrumb, BreadcrumbItem } from '..'

test('should render default Carousel', () => {
  const wrapper = mount(
    <Breadcrumb separator="&gt;">
      <BreadcrumbItem to="/">首页</BreadcrumbItem>
      <BreadcrumbItem to="/">活动管理</BreadcrumbItem>
      <BreadcrumbItem>活动编辑</BreadcrumbItem>
    </Breadcrumb>
  )
  expect(wrapper.html()).toMatchSnapshot()
})
