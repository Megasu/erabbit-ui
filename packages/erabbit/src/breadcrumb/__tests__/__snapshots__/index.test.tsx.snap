// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`should add Transition component use animate 1`] = `
"<div class=\\"er-breadcrumb\\">
  <div class=\\"er-breadcrumb-item\\"><a href=\\"javascript:;\\">首页</a>
    <!---->
  </div>
  <div class=\\"er-breadcrumb-item\\"><a href=\\"javascript:;\\">活动管理</a>
    <!---->
  </div>
  <div class=\\"er-breadcrumb-item\\">活动编辑
    <!---->
  </div>
</div>"
`;

exports[`should render default Carousel 1`] = `
"<div class=\\"er-breadcrumb\\">
  <div class=\\"er-breadcrumb-item\\"><a href=\\"javascript:;\\">首页</a>
    <!---->
  </div>
  <div class=\\"er-breadcrumb-item\\"><a href=\\"javascript:;\\">活动管理</a>
    <!---->
  </div>
  <div class=\\"er-breadcrumb-item\\">活动编辑
    <!---->
  </div>
</div>"
`;
